### ISSUE TITLE
Make messages easier to read by reducing line spacing in message to separate them from other messages.

### ISSUE THREAD

**Issue by bwinton** [vector-im/riot-web#13672]
### Description

I've been thinking a little this morning about why I find messages in the Riot Desktop app harder to read than other apps, and I think it's mostly that the spacing between lines is the same as the spacing between usernames and content, so it's difficult for me to tell which parts go together, and who's saying what.

I made some tweaks, and changed it from:
<img width="797" alt="Screen Shot 2020-05-13 at 10 21 26" src="https://user-images.githubusercontent.com/159571/81949419-d7886d80-95d0-11ea-9e2b-814262f24a78.png">

to:
<img width="798" alt="Screen Shot 2020-05-13 at 10 21 40" src="https://user-images.githubusercontent.com/159571/81950008-71501a80-95d1-11ea-9f69-af0063a65b5a.png">

which I _think_ we can agree makes it clearer what text belongs with which user (particularly on the top two lines and bottom two lines).

Logs being sent: yes

### Version information

- **Platform**: desktop
- **OS**: macOS
- **Version**: Riot Nightly (2020051401)

**Comment by jniniinoinoino** [comment]
Yes! I actually was just about to create my own issue about this. My eyes hurt everytime I try to read from a room where alot is going on. _Especially_ when people **quote** each other, cant even find my own pepe in here. 
![Screenshot from 2020-05-15 16-30-14](https://user-images.githubusercontent.com/61205239/82061826-bbf89200-96b8-11ea-86cb-e2887cdb5c49.png)
